src/Hng.Infrastructure/Context/MyDBContext.cs

```diff
@@ -7,10 +10,15 @@
 
 namespace Hng.Infrastructure.Context
 {
-    public class MyDBContext : DbContext
+    public class MyDBContext : IdentityDbContext<User, IdentityRole<int>, int>
```

Comment:
@davideshett don't hook up to the default ASP.Net user identity roles. 
Just create a simple user table. Custom roles/permissions will be implemented later.